src/main/java/teammates/storage/sqlapi/UsersDb.java

```diff
@@ -292,6 +294,30 @@ public Instructor getInstructorForEmail(String courseId, String userEmail) {
         return HibernateUtil.createQuery(cr).getResultStream().findFirst().orElse(null);
     }
 
+    /**
+     * Gets instructors with the specified {@code userEmail}.
+     */
+    public List<Instructor> getInstructorsForEmail(String courseId, List<String> userEmails) {
```

Comment:
do you mean `getInstructorsForEmails`? since we are getting users for a list of emails, not 1 email

Reply:
I think the convention used is usually `getXByField` or `getXsbyField`, hence I left the Email as singular, but to follow the current TEAMMATES convention I will update to plural form

Reply:
fixed and resolved